plugins/docker/build.ml

```diff
@@ -17,15 +17,15 @@ let use_pool pool f =
 module Key = struct
   type t = {
     commit : [ `No_context | `Git of Current_git.Commit.t ];
-    dockerfile : string option;
+    dockerfile : [`File of string | `Contents of string];
```

Comment:
```suggestion
    dockerfile : [`File of Fpath.t | `Contents of string];
```
Is there a reason not to use `Fpath.t` here? It seems cleaner to me.

Reply:
Good idea; I've updated it.